aging_due_report/wizard/wizard.py

```diff
@@ -585,9 +511,61 @@ def compute_lines(self, partner_ids):
                 aawp_ids[partner_id, currency_id] = aawp_id.id
             line['aawp_id'] = aawp_ids[partner_id, currency_id]
             line['aaw_id'] = wzd_brw.id
+            line['due_days'] = self._get_due_days(today, line)
             aawd_obj.create(line)
+
+        # Only to trigger _get_amount
+        wzd_brw._get_amount()
         return True
 
+    def _get_amount(self):
+        field_sum = ['residual', 'payment', 'total', 'not_due']
+        field_spans = ['span01', 'span02', 'span03', 'span04', 'span05']
+        field_names = field_spans + field_sum
+        direction = self.direction == 'past'
+        spans = [
+            self.period_length * x * (direction and 1 or -1)
+            for x in range(5)]
+
+        for wzd_cur in self.currency_ids:
+            residual = not_due = span01 = span02 = span03 = span04 = 0.0
```

Comment:
You can use directly `_cache` in order to avoid define these local variables
`wiz_cur._cache['residual'] = 0` <- This avoid a write for database
`wiz_cur.residual` <- This avoid a read for database, read directly from the cache

This is similar to https://github.com/odoo/odoo/pull/14005/files#diff-0efb3242408405e7b60f00cdb80f83cfR25